pricing-page-exercise/merve/style.css

```diff
@@ -0,0 +1,277 @@
+* {
+    box-sizing: border-box;
+}
+
+header {
+    width: 100%;
+    margin: 0;
+    display: flex;
+    height: 60px;
+    padding: 0;
+    box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+}
+
+.left {
+    width: 25%;
```

Comment:
Remove width, use flex instead. You can set `justify-content: space-between` to header element to fix sizing and alignment